geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/planning/QueryRunner.scala

```diff
@@ -90,7 +92,17 @@ trait QueryRunner {
     */
   protected [geomesa] def getReturnSft(sft: SimpleFeatureType, hints: Hints): SimpleFeatureType = {
     import org.locationtech.geomesa.index.conf.QueryHints.RichHints
-    hints.getTransformSchema.getOrElse(sft)
+    if (hints.isBinQuery) {
```

Comment:
Does this mean that *all* the Bin/Arrow/Stats/Density processing is handled in the GeoMesa DataStore QueryRunner?

If so, does this change any of the GeoMesa WPS Processes?

Reply:
previously not all our data stores supported all the different query types we do, so this was overridden per data store. but now they all support all our query types, so the logic was consolidated here.
It shouldn't affect any of our wps processes - possibly you will be able to do some wps processes against data stores that you weren't able to before, although most of our processes check to see if the query was handled and if not they do it locally.